source/codegen/metadata/nirfmxspecan/config.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+config = {
+    'api_version': '20.0.0',
+    'c_header': 'niRFmxSpecAn.h',
+    'c_function_prefix': 'RFmxSpecAn_',
+    'service_class_prefix': 'NiRFmxSpecAn',
+    'java_package': 'com.ni.grpc.nirfmxspecan',
+    'csharp_namespace': 'NationalInstruments.Grpc.NiRFmxSpecAn',
+    'namespace_component': 'nirfmxspecan',
+    'close_function': 'Close',
+    'custom_types': [],
+    'additional_headers': {},
+    'type_to_grpc_type': {
+        'niRFmxInstrHandle': 'nidevice_grpc.Session',
+        'char[]': 'string',
```

Comment:
I just took the rest of this from DAQmx, not sure if we'll need all of these or not.